cpp_linter/run.py

```diff
@@ -489,6 +507,23 @@ def run_clang_tidy(
         diff info.
     :param database: The path to the compilation database.
     :param repo_root: The path to the repository root folder.
+    :param extra_args: A list of extra arguments used by clang-tidy as compiler
+        arguments.
+
+        .. note::
+            If the list is only 1 item long and there is a space in the first item,
+            then the list is reformed from splitting the first item by whitespace
+            characters.
+
+            .. code-block:: shell
+
+                cpp-linter -extra-arg="-std=c++14 -Wall"
```

Comment:
```suggestion
                cpp-linter --extra-arg="-std=c++14 -Wall"
```

Reply:
The example code seems miss a `-`

Reply:
Run `cpp-linter --help`, and the output contains the syntax of the document, like `.. code-block:: shell`, I guess it works as designed

```bash
-x EXTRA_ARG, --extra-arg EXTRA_ARG
                        A string of extra arguments passed to clang-tidy for use as
                        compiler arguments. This can be specified more than once for each
                        additional argument. Recommend using quotes around the value and
                        avoid using spaces between name and value (use ``=`` instead):
                        
                        .. code-block:: shell
                        
                            cpp-linter --extra-arg="-std=c++17" --extra-arg="-Wall"
                        
                        Defaults to ``'[]'``.
```

Reply:
Oh I missed this comment.

I'll add it to main.

Reply:
Fixed. Good eye!